full/src/test/java/apoc/trigger/TriggerClusterTest.java

```diff
@@ -104,4 +107,79 @@ public void testTxIdAfterAsync() throws Exception {
         org.neo4j.test.assertion.Assert.assertEventually(() -> TestContainerUtil.<Long>singleResultFirstColumn(cluster.getSession(), "MATCH p = ()-[r:GENERATED]->() RETURN count(p) AS count"),
                 (value) -> value == 2L, 30, TimeUnit.SECONDS);
     }
+
+    //
+    // test cases duplicated, regarding new procedures
+    //
+
+    @Test
+    public void testTimeStampTriggerForUpdatedPropertiesNewProcedures() throws Exception {
+        cluster.getSession().run("CALL apoc.trigger.install('neo4j', 'timestamp','UNWIND apoc.trigger.nodesByLabel($assignedNodeProperties,null) AS n SET n.ts = timestamp()',{})");
+        cluster.getSession().run("CREATE (f:Foo) SET f.foo='bar'");
+        TestContainerUtil.testCall(cluster.getSession(), "MATCH (f:Foo) RETURN f", (row) -> {
+            assertTrue(((Node) row.get("f")).containsKey("ts"));
+        });
+    }
+
+    @Test
+    public void testReplicationNewProcedures() {
+        cluster.getSession().run("CALL apoc.trigger.install('neo4j', 'timestamp','UNWIND apoc.trigger.nodesByLabel($assignedNodeProperties,null) AS n SET n.ts = timestamp()',{})");
+        // Test that the trigger is present in another instance
+        awaitProcedureUpdated(cluster.getDriver().session(), "timestamp");
+    }
+
+    @Test
+    public void testLowerCaseNameNewProcedures() {
+        cluster.getSession().run("create constraint on (p:Person) assert p.id is unique");
+        cluster.getSession().run("CALL apoc.trigger.install('neo4j', 'lowercase', 'UNWIND apoc.trigger.nodesByLabel($assignedLabels,\"Person\") AS n SET n.id = toLower(n.name)',{})");
+
+        awaitProcedureUpdated(cluster.getSession(), "lowercase");
+
+        cluster.getSession().run("CREATE (f:Person {name:'John Doe'})");
```

Comment:
We check the trigger is installed in maybe another instance and then the CREATE could go to a whole different instance? If yes this could lead to another flaky test.